db/db.go

```diff
@@ -0,0 +1,25 @@
+package db
+
+import (
+	"time"
+)
+
+// CronJob is the normalized format interfacing with API handlers
+type CronJob struct {
+	ID       string
+	IsActive bool
+	Function string
+	Workload string
+	CronTime string
+	TimeZone string
+	Created  time.Time
+}
+
+// DB is required type when registering API handlers
+type DB interface {
+	GetDistinctActiveFunctions() ([]string, error)
+	GetJobDetails(job string) ([]CronJob, error)
```

Comment:
param should be called `function`

also something about the name of this function seems off to me still--seems like there should be a plural `Jobs` in the name, and the word `Details` seems superfluous

but I can let it go if you don't feel that way